config.json

```diff
@@ -44,6 +44,7 @@
     "disableStacktrace": false,
     "encoding": "console",
     "outputPaths": [
+      "stdout",
       "goshimmer.log"
     ],
     "disableEvents": false

```

Comment:
the status screen was the only plugin using the logging events. Do we want to disable them here and in the default settings?

Reply:
sure

Reply:
I will use them for #211 
But this new plugin also shouldn't be enabled by default. So I guess it's fine to do that and when the remote log feature is activated this needs enabling too then.